core/circuit_breaker/rule_manager.go

```diff
@@ -64,8 +95,8 @@ func onRuleUpdate(rules []Rule) (err error) {
 	newBreakerRules := make(map[string][]Rule)
 
 	for _, rule := range rules {
-		if !rule.isApplicable() {
-			logger.Warnf("Ignoring invalid breaker rule when loading new rules, %+v.", rule)
+		if err := rule.IsApplicable(); err != nil {
+			logger.Warnf("Ignoring invalid breaker rule when loading new rules, err: %+resRules.", err)
```

Comment:
Maybe it's better to print both err and the bad rule?

Reply:
Okay